.github/workflows/preview.yaml

```
@@ -0,0 +1,49 @@
+name: preview
+
+on:
+  pull_request:
+
+permissions:
+  packages: write
```

```suggestion
  contents: read
  packages: write
```
リポジトリを読み取るのに必要だったりするかも？ 一度 permissions を明示的に書いてしまうと書かれていない権限が全て none になる、らしいです

---

既にイメージビルドCIは成功しているので、必要ないかなと思います。Publicだと必要なかったりするのかな？